error-codes.md

```diff
@@ -451,12 +431,6 @@ In addition, TiDB has the following unique error codes:
 
     Extend the interval of GC Life Time.
 
-* Error Number: 9500
```

Comment:
This change does not show in [pingcap/docs-cn#3160](https://github.com/pingcap/docs-cn/pull/3160).